pkgs/development/tools/analysis/checkov/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, pkgs, lib, python3, fetchFromGitHub }:
+
+let
+  pname = "checkov";
+  version = "1.0.674";
+  src = fetchFromGitHub {
+    owner = "bridgecrewio";
+    repo = pname;
+    rev = version;
+    sha256 = "/S8ic5ZVxA2vd/rjRPX5gslbmnULL7BSx34vgWIsheQ=";
+  };
+
+  disabled = pkgs.python3Packages.pythonOlder "3.7";
+
+  # CheckOV only work with `dpath 1.5.0`
+  dpath = pkgs.python3Packages.buildPythonPackage rec {
+    pname = "dpath";
+    version = "1.5.0";
+
+    src = pkgs.python3Packages.fetchPypi {
+      inherit pname version;
+      sha256 = "SWYVtOqEI20Y4NKGEi3nSGmmDg+H4sfsZ4f/KGxINhs=";
+    };
+
+    doCheck = false;
+  };
+
+  python = pkgs.python3.buildEnv.override {
+    extraLibs = [ dpath ];
+  };
```

Comment:
```suggestion
```

you should use python3Packages everywhere in the file.

Reply:
Sorry but how do I should write python3Packages in L28? I have checked the docs and it seems that `buildEnv` is not in `python3Packages`

Reply:
I don't think you need this buildEnv at all. Add runtime python dependencies like dpath to propagatedBuildInputs.
Anyway, python is at `python3Packages.python` so buildEnv is at `python3Packages.python.buildEnv` I think.